cmd/edit.go

```diff
@@ -64,7 +64,7 @@ func edit(cmd *cobra.Command, args []string) (err error) {
 }
 
 func fileContent(fname string) string {
-	data, _ := os.ReadFile(fname)
+	data, _ := os.ReadFile(config.Expand(fname))
```

Comment:
It's not necessary to put this here actually - we should just make sure that all the places that call fileContent pass in the expanded path.